### ISSUE TITLE
Show etched runes on /block

### ISSUE THREAD

**Issue by casey** [ordinals/ord#3352]
Should be easy, since rune IDs start with the block ID

**Comment by lugondev** [comment]
I dont know how to query by comparing IDs with the block ID.
What method is used to query in redb?

**Comment by casey** [comment]
You'll need to query `RUNE_ID_TO_RUNE_ENTRY` for a range, like this: `RuneId{ block: BLOCK_NUMBER, tx: 0}..RuneId{block: BLOCK_NUMBER + 1, tx: u32::MAX}`.

**Comment by lugondev** [comment]
Instead use `RuneId{ block: BLOCK_NUMBER, tx: 0}..RuneId{block: BLOCK_NUMBER + 1, tx: u32::MAX}`
I use:

```rust
let rune_min = RuneId {
      block: block_height,
      tx: 0,
    };
let rune_max = RuneId {
  block: block_height,
  tx: total_txs_in_block as u32,
};
```
but it's failed. I will retry.

**Comment by lugondev** [comment]
Let's me resolve this issue!

**Comment by cryptoni9n** [comment]
released as part of v0.18.2
